### PR TITLE
Replace hard-coded strings by string resource value (Fixed #1148)

### LINKED ISSUE #1148: Replace hard-coded strings by string resource values
**Actual Behaviour**

Currently, there are a few layout resource files that contain hard-coded strings.

**Expected Behaviour**

The strings should not be hard-coded into the layout resource files. They should be referenced from string resource file strings.xml .

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by arundhati24**
Fixes #1148 

Changes: Replaced hard-coded string with its equivalent string resource value.

### CODE REVIEW

**Comment by arundhati24** [comment]
@chiragw15 Please review this PR. :slightly_smiling_face:

**Comment by arundhati24** [comment]
@chiragw15 I have fixed the failing CIrcle CI builds. It would be great if you could please review the changes? :slightly_smiling_face:

**Comment by arundhati24** [comment]
@akshatnitd @batbrain7 As asked by our mentors, would you please review this PR? :)

File: app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt [hunk 1]
```diff
@@ -59,18 +59,23 @@ class LoginActivityTest {
         Log.d(TAG, "running testUIViewsPresenceOnLoad..")
 
         // checks if email input field is present
+        onView(withId(R.id.email)).perform(scrollTo())
         onView(withId(R.id.email)).check(matches(isDisplayed()))
```
Comment (batbrain7) [inline app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt]: why did you add this?

Comment (arundhati24) [reply app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt]: @batbrain7 While trying to fix failing Circle CI tests, I made this change. Currently, I am going through tests, trying to write unit tests and also trying to fix these errors. So, I hope this will be fixed soon.

**Comment by iamareebjamal** [comment]
I think this is just sorting strings @arundhati24?

**Comment by arundhati24** [comment]
@iamareebjamal Just fixed that. Please have a look.

File: app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt [hunk 2]
```diff
@@ -74,11 +74,9 @@ class LoginActivityTest {
         onView(withId(R.id.customer_server)).check(matches(isDisplayed()))
 
         // checks if sign up button is present
-        onView(withId(R.id.sign_up)).perform(scrollTo())
```
Comment (iamareebjamal) [inline app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt]: These should not be removed

File: app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt [hunk 3]
```diff
@@ -74,11 +74,9 @@ class LoginActivityTest {
...
         onView(withId(R.id.sign_up)).check(matches(isDisplayed()))
 
         // checks if skip button is present
-        onView(withId(R.id.skip)).perform(scrollTo())
```
Comment (iamareebjamal) [inline app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt]: Same here

File: app/src/androidTest/java/org/fossasia/susi/ai/signup/SignUpActivityTest.kt [hunk 4]
```diff
@@ -66,7 +66,6 @@ class SignUpActivityTest {
         onView(withId(R.id.customer_server)).check(matches(isDisplayed()))
 
         // checks if sign up button is present
-        onView(withId(R.id.sign_up)).perform(scrollTo())
```
Comment (iamareebjamal) [inline app/src/androidTest/java/org/fossasia/susi/ai/signup/SignUpActivityTest.kt]: Same here

File: app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt (hunk 2 above)
Comment (arundhati24) [reply app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt]: @iamareebjamal Actually, these statements were not there initially. I had added these because the issue with failing UI tests was "cannot find view with id..." So, as a beginner in testing I was trying to fix it that time. But, the tests are still failing. Should I add these back?

File: app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt (hunk 3 above)
Comment (arundhati24) [reply app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt]: @iamareebjamal Actually, these statements were not there initially. I had added these because the issue with failing UI tests was "cannot find view with id..." So, as a beginner in testing I was trying to fix it that time. But, the tests are still failing. Should I add these back?

File: app/src/androidTest/java/org/fossasia/susi/ai/signup/SignUpActivityTest.kt (hunk 4 above)
Comment (arundhati24) [reply app/src/androidTest/java/org/fossasia/susi/ai/signup/SignUpActivityTest.kt]: @iamareebjamal Actually, these statements were not there initially. I had added these because the issue with failing UI tests was "cannot find view with id..." So, as a beginner in testing I was trying to fix it that time. But, the tests are still failing. Should I add these back?

**Comment by arundhati24** [comment]
@iamareebjamal yes that's right. I think I had sorted the strings for myself to make it easier to add new strings.

**Comment by iamareebjamal** [comment]
I think this should be closed and a new PR should be sent @arundhati24?

**Comment by arundhati24** [comment]
@iamareebjamal Sure. Will do that asap.
